packages/@ember/component/template-only.ts

```diff
@@ -0,0 +1,16 @@
+// This is only exported for types, don't use this class directly
+export class TemplateOnlyComponent {
+  constructor(public moduleName: string) {}
```

Comment:
`moduleName?`

Reply:
Defaulted to `@ember/component/template-only`, and updated the `toString` to `return this.moduleName`.